guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>"precisely segmenting <strong>a </strong>mask for each object instance." (added article 'a')</p><p> </p><p>Another use case is in case of Self Driving Cars -&gt; user lower case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>